AK/Platform.h

```diff
@@ -282,3 +282,5 @@
 #    undef AK_CACHE_ALIGNED
 #endif
 #define AK_CACHE_ALIGNED alignas(AK_SYSTEM_CACHE_ALIGNMENT_SIZE)
+
+#define SERENITY_PAGE_SIZE 4096
```

Comment:
It's used in two places in LibELF, it probably doesn't have to touch such a common header. I'd probably put it somewhere in LibELF for now, but failing that probably its own header in AK.

Do you think it make sense to do the

```
#if __serenity__
#define SERENITY_PAGE_SIZE PAGE_SIZE
#else
#define SERENITY_PAGE_SIZE 4096
#endif
```

Seems vaguely nice to me, but up to you.